R/pkgs/report_generation/tests/testthat/test-test_loading.R

```diff
@@ -124,7 +125,7 @@ test_that("Simulation loading loads the correct number of simulations", {
       num_files = 1
     )
     nrow(a)
-  }, 1 * 10 * 2 * 2)
```

Comment:
This one is actually correct.  The behaviour of the function in case of multiple scenarios with num_files is confusing

Reply:
Ah, I see what you're saying